agentops/time_travel.py

```diff
@@ -64,14 +64,14 @@ def fetch_completion_override_from_time_travel_cache(kwargs):
 def find_cache_hit(prompt_messages, completion_overrides):
     if not isinstance(prompt_messages, (list, tuple)):
         print(
-            "Time Travel Error - unexpected type for prompt_messages. Expected 'list' or 'tuple'. Got ",
+            "🖇️ Agentops: ⏰ Time Travel Error - unexpected type for prompt_messages. Expected 'list' or 'tuple'. Got ",
```

Comment:
Nit: Print prefix is not very DRY. Maybe create a string variable and just do prefix + "message"

Reply:
Marking as resolved bc this is from a different PR and shouldn't block this one

https://github.com/AgentOps-AI/agentops/pull/388